Octokit.Tests/Exceptions/LegalRestrictionExceptionTests.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using Octokit.Internal;
+using Xunit;
+
+namespace Octokit.Tests.Exceptions
+{
+    public class LegalRestrictionExceptionTests
+    {
+        [Fact]
+        public void HasDefaultMessage()
+        {
+            var response = new Response(HttpStatusCode.Forbidden, null, new Dictionary<string, string>(), "application/json");
```

Comment:
No need to apologosie!  Everyone's contributions are much appreciated  :grinning: